src/styles/container.scss

```diff
@@ -0,0 +1,45 @@
+.container {
+  display: grid;
+  grid-template-areas: "first"
+    "second"
+    "third"
+    "fourth"
+    "fifth"
+    "sixth";
+
+  &__block {
+    display: flex;
+    min-width: 300px;
+    height: 300px;
+    color: white;
+    justify-content: center;
+    align-items: center;
+  }
+
+  @each $index, $color in $colors {
+    & :nth-child(#{$index}) {
+      background-color: $color;
+    }
+  }
+
+  @each $index, $number in $numbers {
+    & :nth-child(#{$index}) {
+      grid-area: $number;
+    }

```

Comment:
Maybe you can use this loop to also give each block its number instead of doing so in html

Reply:
As a training of using loops it's fine